src/components/VariableInterpolation.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import ConfigExample from "../../src/components/ConfigExample.tsx";
+
+export const config = {
+	headers: {
+		country: "${.ngrok.geo.country_code}",
+	},
+};
+
+const VariableInterpolation = (type) => {
+	const templateSyntax = "${}";
+
+	return (
+		<>
+			<h3>Variable Interpolation</h3>
+			<p>
+				You may interpolate variables into header values. Variables are
+				interpolated into headers with JSONPath expressions surrounded by{" "}
+				<code>{templateSyntax}</code> syntax.
+			</p>
+			<p>
+				For example, to include geographical data about the client IP that
+				initiated the request, you may construct a header value like so.
+			</p>
+			<ConfigExample config={{ actions: [{ type, config }] }} />
+			<p>
+				Consult the{" "}
+				<a href="/docs/http/request-headers/#variables">Variables Reference</a>{" "}
```

Comment:
when I click the link in the preview it flashes "page not found" then redirects to variable refs

Reply:
oh I was not thinking a link that takes you to the module, I was the variable interpolation documentation on the module page would be broken out into a component that was shared

Reply:
you want me to take all the variables and add them to this component?

Reply:
yiss